sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/models/ImageGenerations.java

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+package com.azure.ai.openai.implementation.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.Immutable;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+
+/** The result of the operation if the operation succeeded. */
+@Immutable
+public final class ImageGenerations {
```

Comment:
This class is the same class as `ImageResponse` and  DataModel Base replaced `ImageLocation` as the `data` type.  Is this intentional?

https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ImageResponse.java#L28

Reply:
Ah. I see. Seems there could be many data types,
`ImageLocationListDataModel extends DataModelBase`

Reply:
Yes, tsp got a union https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/OpenAI.Inference/models/images/images.tsp#L86

That is the reason we had to put this class in implementation package.